h5p_proplist_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package hdf5
```

Comment:
I'd put this test file under the `package hdf5_test`.
AFAICT, the test doesn't use any internals of the `hdf5` package.

Reply:
All test files are put in hdf5 package. Shall we change all of them?

Reply:
New files get their copyright year from the year they are written.
We don't update old files' copyright.

Reply:
It's more of a case by case basis.
(That said, there are perhaps test files that could be put in a hdf5_test pkg. But thus can be part of another PR)